client/app/smallgroup/smallgroup.controller.js

```diff
@@ -10,24 +10,18 @@ angular.module('observatory3App')
 
   var updateSmallGroup = function (callback) {
     callback = callback || function () {};
-
-    if (!$scope.user.smallgroup){
-      $scope.loaded = true;
-      $scope.smallgroup = null;
-      return;
-    }
-
-    $http.get('/api/smallgroup/' + $scope.user.smallgroup).success(function (smallgroup) {
+    return User.smallgroup({id:$scope.user._id})
+    .$promise.then(function(smallgroup){
       $scope.smallgroup = smallgroup;
       $scope.loaded = true;
-      if (!smallgroup) {
+      if (!$scope.smallgroup._id) {
         $scope.smallgroup = false;
```

Comment:
No particular reason, but this is the way it was before, and ensures that the front-end treats it correctly